src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -833,6 +833,62 @@ public void SocketSendReceiveBufferSize_SetZero_ThrowsSocketException()
                 Assert.Equal(e.SocketErrorCode, SocketError.InvalidArgument);
             }
         }
+
+        [Fact]
+        public void SendRecvIovMax_Success()
+        {
+            // sending/receiving more than IOV_MAX segments causes EMSGSIZE on some platforms.
+            // This is handled internally so this error shouldn't surface.
+            const int IOV_MAX = 1024; // Linux
```

Comment:
Rather than tying the value to a specific platform, could we just make it something massive for this purpose, with a comment it should be greater than IOV_MAX on all platforms? Is 1024 the limit on macOS?

Reply:
Looks like it's 1024 on Mac too: https://github.com/opensource-apple/xnu/blob/53c5e2e62fc4182595609153d4b99648da577c39/bsd/sys/uio.h#L244